roborazzi/src/main/java/com/github/takahirom/roborazzi/RobolectricDeviceQualifiers.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.takahirom.roborazzi
+
+object RobolectricDeviceQualifiers {
+  // Generated by [GenerateQualifiersTest.kt]
+  // Data from: AOSP https://android.googlesource.com/platform/tools/base/+/mirror-goog-studio-master-dev/sdklib/src/main/java/com/android/sdklib/devices
+// FROM: tv.xml
+  const val AndroidTV1080p = "w7670dp-h4314dp-xlarge-long-notround-television-xhdpi-keyshidden-dpad"
+  const val AndroidTV720p = "w7670dp-h4314dp-xlarge-long-notround-television-tvdpi-keyshidden-dpad"
+
+  // FROM: devices.xml
+// FROM: wear.xml
+  const val WearOSSquare = "w186dp-h186dp-small-long-round-watch-hdpi-keyshidden-nonav"
+  const val WearOSRound = "w213dp-h213dp-small-long-round-watch-hdpi-keyshidden-nonav"
+  const val WearOSRoundChin = "w249dp-h226dp-small-notlong-notround-watch-tvdpi-keyshidden-nonav"
+
+  // FROM: nexus.xml
```

Comment:
@takahirom 
I've checked with the previous values, and for Pixel4a, height and weight were slightly different as on my device.

The qualifier was:
qualifiers = "w390dp-h844dp-normal-long-440dpi..."

But to make it look like in the phone: qualifiers = "w393dp-h847dp-normal-long-440dpi"

And I believe that is because you calculate width and height from the devices.xml file by using xdpi and ydpi.

If you just use density directly on width and height, you get the values that look the same as on the device, at least on Pixel4a. I cannot confirm on other phones though...

Reply:
Thank you. I didn't notice that. I fixed it by using pixelDensity. For instance, the value for AndroidTV1080p was w7670dp, and now it's w960dp. It seems correct.
https://github.com/takahirom/roborazzi/pull/49/commits/a797fcccd7cf3d5c4ee81cc1ab8c4eb1393e5da0